client/sites/v2/site-settings/controller.tsx

```diff
@@ -1,14 +1,6 @@
-import { isEnabled } from '@automattic/calypso-config';
 import page from '@automattic/calypso-router';
 import type { Context as PageJSContext } from '@automattic/calypso-router';
 
-export function redirectToHostingDashboardBackportIfEnabled(
-	context: PageJSContext,
-	next: () => void
-) {
-	if ( isEnabled( 'dashboard/v2/backport/site-settings' ) ) {
-		return page.redirect( `/sites/${ context.params.site }/settings` );
-	}
-
-	next();
+export function redirectToHostingDashboardBackportIfEnabled( context: PageJSContext ) {
```

Comment:
Since the flag is being removed, let's change it to:
```suggestion
export function redirectToHostingDashboardBackport( context: PageJSContext ) {
```

Reply:
The flag removed by this PR is `dashboard/v2/backport/site-settings` 😄